### PR TITLE
refactor rss

### LINKED ISSUE #126: refactor rss
## Description
change Rss codes with np.vectorize() 
## Check list 
- [ ]

### PR DESCRIPTION
**Pull request by kaikai529**
## Description

refactor rss
## Related Issues and Reference

Close #126 
## Test

merge #125 and write test

### CODE REVIEW

**Review by Hattomo — COMMENTED** [review]
Look OSS code!

- 
  Flutter https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/material.dart

- VScode https://github.com/microsoft/vscode/blob/master/src/main.js

There are many comments on it that you can understand easily.  
I think deleting comments **Immoderately** makes us inefficient, because we forget easily what we did. :neutral_face:

**Comment by kaikai529** [comment]
Ok. We use comment actively. But I don’t know where comment is needed. So please make simple comment rule.

File: tests/test_costfunction.py [hunk 1]
```diff
@@ -22,5 +30,4 @@ def test_diffcross_entropy():
     assert np.isclose(error, np.array([1.250, 1.428, -2.500, -2.000]), rtol=1e-3,
                       atol=1e-3).all(), "check diffcross_entropy or test"
 
-test_cross_entropy()
-test_diffcross_entropy()
\ No newline at end of file
+test_diffrss()
```
Comment (Hattomo) [inline tests/test_costfunction.py]: you don't have to run this
```suggestion

```

**Review by Hattomo — COMMENTED** [review]
Nits

**Comment by kaikai529** [comment]
what is nits ?

**Review by Hattomo — APPROVED** [review]
thank you for your hard work!

**Comment by Hattomo** [comment]
> what is nits ?

Nits mean "些細なことだけど"

**Pull request merged**